MTM-49511 revoke tokens documentation (#786)
SoftwareAG/c8y-docs: content/users-guide/getting-started-bundle/user-settings.md

Note that this option is only available if support user access is not set globally for subtenant users in the {{< management-tenant >}}, for details see <a href="/users-guide/enterprise-tenant/#support-user-access" class="no-ajaxy">Support user access</a>. After the support user access has been activated, the menu item switches to <strong>Deactivate support</strong>. If your support request has been resolved but the duration for the support user access is not expired (24 hours per default) you can actively disable an active support user request here.</td>
</tr>
<tr>
<td style="text-align:left">Revoke tokens</td>

> <td style="text-align:left">Revoking tokens will log out all users logged with "OAI-Secure" or "Single sign-on redirect". Note that also JWT tokens retrieved by device will be revoked.<br>

Reviewer: ```suggestion
<td style="text-align:left">Revoking tokens logs out all users currently logged in via OAI-Secure or single sign-on redirect. Note that JWT tokens retrieved by devices are also revoked.<br>
```

> <td style="text-align:left">Revoking tokens will log out all users logged with "OAI-Secure" or "Single sign-on redirect". Note that also JWT tokens retrieved by device will be revoked.<br>

Reviewer 2: ```suggestion
<td style="text-align:left">Revoking tokens will log out all users logged in with "OAI-Secure" or "Single sign-on redirect". Note that the JWT tokens retrieved by a device will be revoked as well.<br>
```

> <td style="text-align:left">Revoking tokens will log out all users logged in with "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.<br>

Reviewer 2: ```suggestion
<td style="text-align:left">Revoking tokens logs out all users currently logged in via "OAI-Secure" or "Single sign-on redirect". Note that JWT tokens retrieved by your devices will also be revoked.<br>
```

<br>

> To revoke tokens, the user must have the permission ROLE_USER_MANAGEMENT_ADMIN.

Reviewer 2: ```suggestion
To revoke tokens, the user must have ADMIN permission for the permission type "User management".  
```

> To revoke tokens, the user must have the permission ROLE_USER_MANAGEMENT_ADMIN.

Reviewer: ```suggestion
To revoke tokens, you need ROLE_USER_MANAGEMENT_ADMIN permission.
```

> To revoke tokens, the user must have ADMIN permission for the permission type "User management".  

Reviewer 2: ```suggestion
To revoke tokens, you must have ADMIN permission for the permission type "User management".  
```
